src/Filter/CallbackFilter.php

```diff
@@ -29,7 +29,18 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $value
             throw new \RuntimeException(sprintf('Please provide a valid callback option "filter" for field "%s"', $this->getName()));
         }
 
-        $this->active = \call_user_func($this->getOption('callback'), $queryBuilder, $alias, $field, $value);
+        $isActive = \call_user_func($this->getOption('callback'), $queryBuilder, $alias, $field, $value);
```

Comment:
I know this is not directly related to this PR, but IMO we should add a test case using an array callback.

Reply:
Like this
```
$filter->initialize('field_name', [
    'callback' => [$this, 'customCallback'],
]);
```
?

There is already one.

Reply:
Great! Didn't see it before.